corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -428,7 +430,26 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         events: CaseListViewEvents(),
 
         handleSmallScreenChange: function (enabled) {
-            this.smallScreenEnabled = enabled;
+            const self = this;
+            self.smallScreenEnabled = enabled;
+            if (self.options.sidebarEnabled) {
+                self.positionStickyItems(enabled);
+            }
+        },
+
+        positionStickyItems: function (smallScreenEnabled) {
+            const $caseListHeader = $('#case-list-menu-header');
+            const $caseListMap = $('#module-case-list-map');
+            const stickyHeaderId = 'small-screen-sticky-header';
```

Comment:
One of the uses doesn't have the `#` because it's being used directly as an html element id.

```javascript
$caseListHeader.wrap(`<div class="sticky sticky-header" id="${stickyHeaderId}"></div>`);
```